mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -89,6 +101,14 @@ void CGameMode_RJ::OnPlayerSpawn(CMomentumPlayer *pPlayer)
 #endif
 }
 
+void CGameMode_SJ::OnPlayerSpawn(CMomentumPlayer *pPlayer)
+{
+#ifdef GAME_DLL
+    pPlayer->GiveNamedItem("weapon_momentum_stickylauncher");
+    //pPlayer->GiveNamedItem("weapon_momentum_grenadelauncher"); //TODO: Add Grenade Launcher
```

Comment:
We aren't adding the pipebomb launcher. You can give the shotgun, or if you prefer the meme, the pistol